static/css/disease.scss

```diff
@@ -14,4 +14,31 @@
  * limitations under the License.
  */
 
- @import "base";
\ No newline at end of file
+ @import "base";
```

Comment:
i assume these are similar to the styling for protein. please share these as well, so they can stay consistent with each other.

Reply:
Correct! I'll share them in the next PR, devoted to refactoring. Thanks for pointing it out!